torch/_inductor/codegen/cpp_micro_gemm.py

```diff
@@ -59,13 +59,21 @@ def __init__(
         self.alpha = alpha
 
     def get_common_options(self):
+        if self.input_dtype == torch.uint8:
+            assert self.compute_dtype == torch.int32
+            assert self.output_dtype == torch.int32
         return {
             "torch": torch,
             "kernel_name": self.name,
             "input_dtype": self.input_dtype,
             "output_dtype": self.output_dtype,
             "compute_dtype": self.compute_dtype,
             "input_t": DTYPE_TO_CPP[self.input_dtype],
+            "input2_t": DTYPE_TO_CPP[torch.int8]
+            if self.input_dtype == torch.uint8
+            else DTYPE_TO_CPP[
+                self.input_dtype
+            ],  # TODO: support dtype other than s8 for weight
```

Comment:
Maybe it is better to allow the caller to specify the dtype for `input2` while not hard-code things here? Can extend in future PRs.